rust/processor/src/models/token_v2_models/v2_token_ownerships.rs

```diff
@@ -296,20 +339,19 @@ impl TokenOwnershipV2 {
         query_retries: u32,
         query_retry_delay_ms: u64,
     ) -> anyhow::Result<Option<(Self, CurrentTokenOwnershipV2)>> {
-        let token_address = standardize_address(&write_resource.address.to_string());
-        if let Some(burn_event) = tokens_burned.get(&token_address) {
+        if let Some(burn_event) = tokens_burned.get(token_address) {
```

Comment:
Maybe call `standardize_address()` on `tokens_burned` here too in case we use the helper fn anywhere else in the future

Reply:
ok yea good poitn.